### PR TITLE
Update binary-deployment.md

### PR DESCRIPTION
**Pull request by lilin90**
PTAL

### CODE REVIEW

File: op-guide/binary-deployment.md [hunk 1]
```diff
@@ -1,83 +1,197 @@
...
-To try TiDB out and explore the features, see [Multiple Nodes Cluster Deployment for Test](#multiple-nodes-cluster-deployment-for-test).
+- To quickly understand and try TiDB, see [Single Node Cluster Deployment](#single-node-cluster-deployment).
+- To try TiDB out and explore the features, see [Multiple Nodes Cluster Deployment for Test](#multiple-nodes-cluster-deployment-for-test).
+- To deploy and use TiDB in production, see [Multiple Nodes Cluster Deployment](#multiple-nodes-cluster-deployment).
 
-To deploy and use TiDB in production, see [Multiple Nodes Cluster Deployment](#multiple-nodes-cluster-deployment).
+## TiDB Components and Default Ports
 
-## Download and Decompress the Official Binary Package
+### 1. TiDB Database Components (Required)
```
Comment (andelf) [inline op-guide/binary-deployment.md]: Is 1. Redundant?

File: op-guide/binary-deployment.md [hunk 2]
```diff
@@ -1,83 +1,197 @@
...
+| Component | Default Port | Protocol | Description | 
+| :-- | :-- | :-- | :----------- |
+| ssh | 22 | TCP | sshd service |
+| TiDB|  4000  | TCP | the communication port for the application and DBA tools | 
+| TiDB| 10080  |  TCP | the communication port to report TiDB status | 
+| TiKV|  20160 |  TCP | the TiKV communication port  | 
+| PD | 2379 | TCP | the communication port between TiDB and PD | 
+| PD | 2380 | TCP | the inter-node communication port within the PD cluster |
+
+### 2. TiDB Database Components (Optional)
```
Comment (andelf) [inline op-guide/binary-deployment.md]: Structurally wrong title. Is it monitoring or sth.?

File: op-guide/binary-deployment.md [hunk 3]
```diff
@@ -1,83 +1,197 @@
...
 category: deployment
 ---
 
-# Binary Deployment
+# Deploy TiDB Using the Binary
 
 ## Overview
 
-A complete TiDB project contains PD, TiKV, TiDB. The start-up sequence is PD -> TiKV -> TiDB.
+A complete TiDB cluster contains PD, TiKV and TiDB. To start the database service, follow the order of PD -> TiKV -> TiDB. To close the database service, follow the reverse order of start-up.
```
Comment (queenypingcap) [inline op-guide/binary-deployment.md]: To close the database service, follow the reverse order of start-up.
->
To stop the database service, follow the order of stopping TiDB -> TiKV -> PD.

File: op-guide/binary-deployment.md (hunk 1 above)
Comment (queenypingcap) [reply op-guide/binary-deployment.md]: Let's remove all the numbers from the titles

**Pull request merged**